scripts/install-netdata.sh

```
@@ -0,0 +1,363 @@
+# =============================================================================
+# Claim Nodes to Cloud
+# =============================================================================
+
+claim_nodes() {
+    log_info "Claiming NetData nodes to NetData Cloud..."
+    
+    # Check for claim token
+    if [[ -z "${NETDATA_CLAIM_TOKEN:-}" ]]; then
+        # Try to load from .env
+        if [[ -f "$PROJECT_DIR/.env" ]]; then
+            source "$PROJECT_DIR/.env"
```

Shellcheck (0.11.0)

[info] Not following: ./.env was not specified as input (see shellcheck -x).

(SC1091)